src/Elcodi/Store/PaymentBridgeBundle/Services/PaymentBridge.php

```diff
@@ -267,6 +270,15 @@ public function getExtraData()
                 $extraData['items'][$orderLine->getId()] = $orderLineArray;
             }
 
+            if ($this->order->getShippingAmount()->isGreaterThan(Money::create(0, $currency))) {
```

Comment:
I would add a comment saying that we are adding shipping costs as a "shadow" product in the `extra_data` structure